.github/workflows/build.yml

```diff
@@ -298,11 +298,14 @@ jobs:
           path: ${{ env.CACHED_BUILD_PATHS }}
           key: ${{ env.BUILD_CACHE_KEY }}
       - name: Check bundle sizes
-        uses: getsentry/size-limit-action@main-skip-step
+        uses: getsentry/size-limit-action@runForBranch
```

Comment:
We'll need to change this once the PR is merged and the branch is deleted, right?

Reply:
jup, although the previous state was also still using the branch 😅 but yes, def should do a new "release" then. But I'll wait on billy to review there first.